src/components/Tooltip/index.js

```diff
@@ -42,6 +42,10 @@ class Tooltip extends PureComponent {
         this.measureTooltip = this.measureTooltip.bind(this);
         this.showTooltip = this.showTooltip.bind(this);
         this.hideTooltip = this.hideTooltip.bind(this);
+        this.showTooltipWithDelay = _.debounce(this.showTooltip.bind(this), this.props.showDelay);
+        this.hideTooltipWithDelay = _.debounce(this.hideTooltip.bind(this), this.props.hideDelay);
```

Comment:
Can we use `Animated.Timing` built in delay config instead of a debounce?
https://reactnative.dev/docs/animated#timing

Reply:
Ah. Nice